jobs/bbr-atcdb/templates/config.json.erb

```diff
@@ -35,8 +35,16 @@
       postgres_host = l.instances.first.address
     end if postgres_host.empty?
 
+    if_link("concourse_db") do |cdb|
+      postgres_host = cdb.p("postgressql.host", "")
+      postgres_port = cdb.p("postgressql.port")
+      postgres_role_name = cdb.p("postgressql.role.name")
+      postgres_role_password = cdb.p("postgressql.role.password")
+      postgres_database = cdb.p("postgressql.database")
+    end if postgres_host.empty?
```

Comment:
In the case of the database having TLS configure, where you must provide certs/etc, this would fail, right? 🤔

I'm wondering about a setup like `hush-house` where the client must connect using those. 

Thanks!

Reply:
We added TLS properties.